### PR TITLE
Adding 'pyxel-sim'

### PR DESCRIPTION
**Pull request by flemmel**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by flemmel** [comment]
Dear maintainers,

This is the first time that I submit a package to 'conda-forge'.

All checks have passed and my branch has no conflicts with the base branch.
Do I have more steps to initiate ?

Thanks !

File: recipes/pyxel-sim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+about:
+  home: https://esa.gitlab.io/pyxel/
+  summary: Pyxel detector simulation framework.
+  dev_url: https://gitlab.com/esa/pyxel
+  license: MIT
+  license_file: LICENSE.txt
+
+extra:
+  recipe-maintainers:
+    - FredericLemmel
```
Comment (wolfv) [inline recipes/pyxel-sim/meta.yaml:72]: you need to use your _github_username here (I guess `flemmel`?)

Looks good otherwise!

**Review by wolfv — CHANGES_REQUESTED** [review]

File: recipes/pyxel-sim/meta.yaml (hunk 1 above)
Comment (flemmel) [reply recipes/pyxel-sim/meta.yaml]: You're right ! 

Now, I am using my GitHub username for the 'recipe-maintainers'.

Thanks !

**Review by mwcraig — APPROVED** [review]

**Comment by mwcraig** [comment]
Thanks, @flemmel, merging!

**Pull request merged**